src/crypto/algorithms/megolm.js

```diff
@@ -840,6 +848,15 @@ MegolmDecryption.prototype.onRoomKeyEvent = function(event) {
 
         // have another go at decrypting events sent with this session.
         this._retryDecryption(senderKey, sessionId);
+    }).then(() => {
+        if (this._crypto.backupInfo) {
+            // don't wait for it to complete
```

Comment:
'it' being the promise returned by backupGroupSession? should prolly spell 'it' out